engine/dlib/src/dmsdk/dlib/buffer.h

```diff
@@ -91,6 +91,8 @@ namespace dmBuffer
      * @member dmBuffer::RESULT_STREAM_MISSING
      * @member dmBuffer::RESULT_STREAM_TYPE_MISMATCH
      * @member dmBuffer::RESULT_STREAM_COUNT_MISMATCH
+     * @member dmBuffer::RESULT_METADATA_INVALID
+     * @member dmBuffer::RESULT_METADATA_NOT_EXIST
```

Comment:
I wonder if this should be named METADATA_MISSING (like STREAM_MISSING above)?

Reply:
It makes sense. Note, I've already merged the 'RESULT_METADATA_NOT_EXIST' in code. It's just the docs that are updated here.

Reply:
Ah, I see. There's still time to change it though. I don't think anyone is using that functionality yet.

Reply:
cool. Let me change it then.